packages/ethereum-storage/package.json

```diff
@@ -52,13 +52,13 @@
     "qs": "6.10.3",
     "shelljs": "0.8.5",
     "tslib": "2.3.1",
-    "web3-eth": "1.3.6",
-    "web3-utils": "1.3.6",
+    "web3-eth": "1.8.1",
+    "web3-utils": "1.8.1",
     "yargs": "16.2.0"
   },
   "devDependencies": {
     "@openzeppelin/test-helpers": "0.5.6",
-    "@truffle/hdwallet-provider": "1.2.3",
+    "@truffle/hdwallet-provider": "2.1.1",
```

Comment:
Hey @mliu 
Just adding this to my comments from our Thursday call

Can you please revert the web3-eth, web3-utils, and hdwallet-provider dependencies changes if they're not required for this PR?

Reply:
will do